plugins/modules/system/xfconf.py

```diff
@@ -86,29 +86,32 @@
 RETURN = '''
   channel:
     description: The channel specified in the module parameters
-    returned: success
+    returned: always
```

Comment:
I don't think that's true. If the module fails, it generally does *not* return this: https://github.com/ansible-collections/community.general/blob/844749f475b61513accb32dee4fcedede335a1b3/plugins/module_utils/module_helper.py#L108